.tasks

```diff
@@ -13,8 +13,12 @@
 
 set -ex
 
-# Scan for all tests
-bots/tests-scan
+# Consume item from task queue or scan for all tests
+if [ -n "$AMQP_URL" ]; then
+    echo bots/run-queue --amqp "$AMQP_URL"
```

Comment:
Given that it's now "host" or "host:port", perhaps the variable should be changed to `$AMQP_SERVER`?

Reply:
It should always be host:port; would `AMQP_HOST_PORT` be better?

Reply:
Sure. When I wrote that,  I didn't realize yet that the port was not optional.